exercises/spiral-matrix/src/example/java/SpiralMatrixBuilder.java

```diff
@@ -0,0 +1,27 @@
+class SpiralMatrixBuilder {
+
+    int[][] ofSize(int size) {
+        if (size == 0) return new int[][]{};
```

Comment:
I would add curly brackets and put the `if` body on a new line, just to be safe :)